src/components/AppLayout/index.tsx

```diff
@@ -122,7 +115,7 @@ const Layout: React.FC<Props> = ({
         </SidebarWrapper>
         <ContentWrapper>
           <div>{children}</div>
-          {hasFooter && <Footer />}
+          <Footer />
```

Comment:
I thought we wanted to hide the footer outside of the welcome page?

Reply:
We need it visible on all pages to expose the AppStore badge to as many users as possible.
Also seeing the app version at all times is very helpful.

@lukasschor you mind if we restore the footer?

Reply:
I am in general unsure on the value of displaying the App Store badge like this. It feels unnatural and I would prefer to promote the mobile app in parts of the user flows where it is actually relevant (setting up a Safe, connecting a wallet) rather than just promoting it randomly in the footer. 

There were some thoughts in not showing the footer everywhere (more screen estate, stronger focus on relevant content). If at all, I would suggest first displaying the app store button in the welcome screen in the footer only and track over some some time the effectiveness of this and how it compares to promoting the app in other places.

But even then, it might be worth exploring other places in the welcome page or onboarding flow where this would fit better.

Reply:
Let's consider this a temporary test until the next release then. We'll be able to gauge how many users will click on this with the footer visible on every screen.
I've made an issue to revert it here: #3712.